client/src/gamedata/en/descriptions/levels/safetoken.md

```diff
@@ -0,0 +1,10 @@
+"I've asked the development team to create and deploy an ERC20 token for me, and this is the source code they've provided. For some reason, the block explorer does not verify the source code. The team has confirmed that it should not be an issue, and everything seems to work just fine anyway."
+
+Find the backdoor and complete the level by draining the owner's account.
+
+
+Things that might help:
+* An [EVM bytecode decompiler](https://ropsten.etherscan.io/bytecode-decompiler?a=) 
```

Comment:
Ropsten is deprecated, maybe is worth changing it for some other tool ? or the same from etherscan but in another chain

Reply:
Yes sure, I'll look at that.